client/components/legacy-species-selector.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, Fragment } from 'react';
+import Field from './field';
+import LEGACY_SPECIES from '../constants/legacy-species';
+
+export default function LegacySpeciesSelector(props) {
+  const otherSpeciesId = LEGACY_SPECIES.find(s => s.label === 'Other species').value;
```

Comment:
Given that the id is immutable but the label is not, it's probably better to hard-code the id than the label.

Reply:
```suggestion
  const otherSpeciesId = 28;
```